### DISCUSSION TITLE
Plans for Availability via Debian Packages

### DISCUSSION THREAD

**Discussion by DarioS** [Chaste/Chaste#331] [Q&A] [answered]
Any plans for https://packages.debian.org/stable/science/ ?

**Comment by jmpf** [comment]
I believe that there are no plans to add Chaste to the science repo.  @fcooper8472 may know better.

**Comment by fcooper8472** [comment, marked as answer]
Thanks @DarioS for your question.

When someone wants to use Chaste to do new science, it is almost inevitable that they have to modify or extend existing functionality. For that reason, most users would either:

1. Create a [user project](https://github.com/Chaste/template_project) which would sit in the Chaste source tree
2. Create a new branch and modify existing classes

In both of these scenarios, the user would compile Chaste from source. So in our view there's fairly limited utility in having pre-compiled libraries available for consumption, and therefore we don't think it would be very helpful for Chaste to be included in the science repo.

If you have a compelling use-case for this, though, please let us know.
